extensions/amp-accordion/0.1/amp-accordion.css

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-accordion [i-amphtml-measure] {
+  /* Need to use CSS (not inline style) to use !important. Overrides a rule
```

Comment:
you can use `setImportantStyles()` from `styles.js` in JS

Reply:
!!!
TIL

Reply:
Neat! Created #17473 to use `setImportantStyles` instead.